tests/test_pygtracker.py

```diff
@@ -22,6 +33,34 @@ def test_version():
 
 # Start tests for rank_students
 
+# tests that the user inputs the right number of students
+def test_number_of_students():
+    assert (
+        gradetracker.rank_students(n=4).shape[0]
+    ) == 4, "Number of students don't match"
+
+
+# raises an error when a course id is not a part of the dataset
+def test_course_input():
+    with raises(NameError):
+        gradetracker.rank_students(course_id="500", n=4)
+
+
+# dataframe is equal to an example dataframe
+def test_df_equal():
```

Comment:
the test needs to have the real output to compare against. It is okay to write just a few tests first (that you expect to run and give you the output that matches what you expected)